lib/vanagon/platform/windows.rb

```diff
@@ -32,6 +32,7 @@ def package_name(project)
       end
 
       # Method to generate the files required to build a windows package for the project
+      # Both the MSI and nuget generic articfacts are created for simplicity.
```

Comment:
This comment isn't necessary. It restates what that code already shows and adds no additional information.